scalopus_python/scalopus/scalopus_python_lib.py

```diff
@@ -3,15 +3,22 @@
 # that's found inside the .egg file instead of 'this' python file.
 def __bootstrap__():
     global __bootstrap__, __loader__, __file__
-    import sys, pkg_resources, imp, sysconfig
+    import sys, pkg_resources, importlib, importlib.machinery, sysconfig
     # Use the ABI suffix, this is "" for python2, for python3 this call returns something like:
     # ['cpython-36m-x86_64-linux-gnu']
     so_abi_suffix = sysconfig.get_config_vars("SOABI")
     so_suffix = ""
     if so_abi_suffix:
         if len(so_abi_suffix):
             so_suffix = so_abi_suffix[0] + "." if so_abi_suffix[0] is not None else ""
-    __file__ = pkg_resources.resource_filename(__name__,'../scalopus_python_lib.{}so'.format(so_suffix))
+    name = __name__
+    filename = './scalopus_python_lib.{}so'.format(so_suffix)
```

Comment:
Would this work here?
```suggestion
    filename = f'./scalopus_python_lib.{so_suffix}so'
```

Reply:
Changed :+1: